enumerables.rb

```diff
@@ -0,0 +1,143 @@
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield (self[i])
+      i += 1
+    end
+  end
+
+  def my_each_with_index
+    i = 0
+    while i < length
+      yield self[i], i
+      i += 1
+    end
+  end
+
+  def my_select
+    selected = []
+    my_each { |i| selected << i if yield(i) }
+    selected
+  end
+
+  def my_all?
+    result = true
+    my_each { |i| result = false unless yield(i) }
+    result
+  end
+
+  def my_any?
+    result = false
```

Comment:
* [ ] There should be working `my_any?` identical and returns the same thing as ruby's [`any?`](https://ruby-doc.org/core-2.6.4/Enumerable.html#method-i-any-3F). ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/advanced_building_blocks_enumerable/my_any.png) from [Odin](https://www.theodinproject.com/courses/ruby-programming/lessons/advanced-building-blocks#assignment-2))
#### Let us compare the results return in each of the examples below and make the return the same result as the original method.
1. ```p %w[ant bear cat].any?(/d/)``` and ```p %w[ant bear cat].my_any?(/d/)``` must return same result.
2. ```p [nil, true, 99].any?(Integer)``` and ```p [nil, true, 99].my_any?(Integer)``` must return same result.
3. ```p [nil, true, 99].any? ``` and ```p [nil, true, 99].my_any? ``` must return same result.